Resources/Locale/en-US/_NF/chat/managers/chat_manager.ftl

```diff
@@ -1,4 +1,9 @@
 chat-speech-verb-vulpkanin-1 = rawrs
 chat-speech-verb-vulpkanin-2 = barks
 chat-speech-verb-vulpkanin-3 = rurs
-chat-speech-verb-vulpkanin-4 = yeeps
```

Comment:
why?

Reply:
DeltaV
Just trying to keep it the same

Reply:
Il honestly prefer to keep it, just add it as a number five.

Reply:
ok..